src/rebuild/srv.c

```diff
@@ -956,8 +1029,12 @@ rebuild_try_merge_tgts(const uuid_t pool_uuid, uint32_t map_ver,
 		found->dst_map_ver = map_ver;
```

Comment:
Not from this patch, but in the "if" statement, the variable is "task", but this line is using "found", I think they are the same, but it's still better to use the same variable?

Reply:
task and found were only the same with the "break" statement. I renamed found to "merge_task" and replaced references to "task" with "merge_task".